tests/lazy-loading.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { test, expect, Request } from '@playwright/test';
+import { openPlot } from './utils';
+
+/**
+ * Checks the size of the response body of a request
+ * If the response body size is 0, the request is not checked
+ * This seems to happen for Safari sometimes
+ *
+ * @param request The request object. Playwright provides size on the request, not the response
+ * @param size The minimum size in bytes
+ */
+async function expectResponseSize(request: Request | undefined, size: number) {
```

Comment:
I missed this before. It would be good to name this something that indicates it is the minimum response size it checks for `expecteMinimumResponseSize` or something like that.